app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+
+  before_action :authorize
+
+  def create
+    @video = Video.find(params[:video_id])
+    review = Review.new(review_params)
+    if review.save
+      redirect_to @video
+    else
+      @reviews = @video.reviews
+      render 'videos/show'
+    end
+  end
+
+  private
+
+  def review_params
+    p = params.require(:review).permit(:rating, :body)
+    p = p.merge(user: current_user)
```

Comment:
The argument is a hash, you can combine this line and the next one in a single hash argument. You also don't need the last `p` assignment, since it is just returning the value anyway. To take it one step further, you can optionally tack the `merge` onto your first line and remove the `p` variable altogether.